extra_scripts/datasets/create_resisc45_data_files.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+import random
+import shutil
+
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+
+
+def get_argument_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-i",
+        "--input",
+        type=str,
+        help="Path to the expanded NWPU-RESISC45.rar archive (download from: https://1drv.ms/u/s!AmgKYzARBl5ca3HNaHIlzp_IXjs)",
+    )
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        help="Folder where the classification dataset will be written",
+    )
+    return parser
+
+
+class _RESISC45:
+    """
+    Dataset used to parallelize the transformation of the dataset via a DataLoader
+    """
+
+    TRAIN_SPLIT_PERCENT = .8
+    TEST_SPLIT_PERCENT = .2
+
+    def __init__(self, input_path: str, output_path: str, train: bool):
+        self.input_path = input_path
+        self.output_path = output_path
+        self.train = train
+        self.images = []
+        self.targets = []
+        self.labels = sorted(os.listdir(self.input_path))
+        random.seed(42)
```

Comment:
As for EuroSat, I suggest having an explicit `random.Random` generator for the split creation (to make it more explicit and resilient to changes)